pennylane/io.py

```diff
@@ -410,6 +410,67 @@ def from_qiskit_op(qiskit_op, params=None, wires=None):
         raise RuntimeError(_MISSING_QISKIT_PLUGIN_MESSAGE) from e
 
 
+def from_qiskit_noise(noise_model, **kwargs):
+    """Converts a Qiskit `NoiseModel <https://qiskit.github.io/qiskit-aer/stubs/qiskit_aer.noise.NoiseModel.html>`__
+    into a PennyLane :class:`~.NoiseModel`.
+
+    .. note::
+
+        This function depends upon the PennyLane-Qiskit plugin. Follow the
+        `installation instructions <https://docs.pennylane.ai/projects/qiskit/en/latest/installation.html>`__
+        to get up and running. You may need to restart your kernel if you are running in a notebook
+        environment.
+
+    Args:
+        noise_model (qiskit_aer.noise.NoiseModel): a ``NoiseModel`` constructed in Qiskit
+        kwargs: Optional keyword arguments for conversion of the noise model.
+
+    Keyword Arguments:
+        kraus_shape (bool): use shape of the Kraus operators to display ``qml.QubitChannel``
+            instead of the complete list of matrices. Default is ``True``.
+        decimals (int): number of decimal places to round the Kraus matrices. Default is ``10``.
+
+    Returns:
+        qml.NoiseModel: An equivalent noise model constructed in PennyLane
+
+    Raises:
+        ValueError: When an encountered quantum error cannoted be converted.
+
+    .. note::
+
+        Currently, PennyLane noise models does not support readout errors, so those will be skipped during conversion.
+
+    **Example**
+
+    Consider the following noise model constructed in Qiskit:
+
+    >>> import qiskit.providers.aer.noise as noise
+    >>> error_1 = noise.depolarizing_error(0.001, 1) # 1-qubit noise
+    >>> error_2 = noise.depolarizing_error(0.01, 2) # 2-qubit noise
+    >>> noise_model = noise.NoiseModel()
+    >>> noise_model.add_all_qubit_quantum_error(error_1, ['rz', 'ry'])
+    >>> noise_model.add_all_qubit_quantum_error(error_2, ['cx'])
+    >>> load_noise_model(noise_model)
+    NoiseModel({
+        OpIn(['RZ', 'RY']): QubitChannel(Klist=Tensor(4, 4, 4))
+        OpIn(['CNOT']): QubitChannel(Klist=Tensor(16, 4, 4))
+    })
+
+    This noise model can be converted into PennyLane using:
+
+    >>> load_noise_model(noise_model)
+    NoiseModel({
+        OpIn(['RZ', 'RY']): QubitChannel(Klist=Tensor(4, 4, 4))
+        OpIn(['CNOT']): QubitChannel(Klist=Tensor(16, 4, 4))
+    })
```

Comment:
```suggestion
    This noise model can be converted into PennyLane using:

    >>> load_noise_model(noise_model)
    NoiseModel({
        OpIn(['RZ', 'RY']): QubitChannel(Klist=Tensor(4, 4, 4))
        OpIn(['CNOT']): QubitChannel(Klist=Tensor(16, 4, 4))
    })
```

Seems like it's repeated.

Reply:
Or, like it's kind of confusing? Aren't you supposed to use `from_qiskit_noise` instead?

Reply:
Nice catch! Yes, it should be `from_qiskit_noise`.